addons/website_crm_partner_assign/static/src/scss/crm_partner_assign.scss

```diff
@@ -0,0 +1,39 @@
+.short_description {
+    max-height: 100px;
+    display: -webkit-box;
+    -webkit-line-clamp: 4;
+    -webkit-box-orient: vertical;
+}
+
+.partner_details_contact address > div div {
+    padding: map-get($spacers, 1) 0;
+
+    &:last-child{
+        border: none;
+    }
+}
+
+.filters_offcanvas {
+    width: 80%;
+    max-width: $offcanvas-horizontal-width;
+
+    & .accordion-button {
```

Comment:
All these customizations mirror `website_sale` ones, am I right? 
Better remove customizations from both modules and act on default BS variables instead.
For properties that can't be customised using variables, write shared SCSS in `website.scss`

Reply:
see https://github.com/odoo/odoo/pull/109752/commits/5e49af5669756538c7dea0b23f824fd922911b93